src/types/block.rs

```diff
@@ -98,6 +103,10 @@ pub struct Block<TX> {
     pub transactions: Vec<TX>,
     /// Size in bytes
     pub size: Option<U256>,
+    #[serde(rename = "mixHash")]
```

Comment:
Missing docs?

Reply:
Sorry, I'll adjust in a minute.

Reply:
Added the missing docstring and changed the type of `nonce` to H64. Here is a reference that supports H64 as the correct type: https://github.com/ethereum/go-ethereum/blob/461291882edce0ac4a28f64c4e8725b7f57cbeae/core/types/block.go:43. Feel free to double check.

Reply:
Fixing the unit test that I broke...